### PR TITLE
WIP - evaluate schema drift

### PR DESCRIPTION
**Pull request by jonallured**
I noticed some drift in the schema that vortex has checked into its repo vs the one that exists in MP so I asked about it here:

https://artsy.slack.com/archives/CP9P4KR35/p1644856104672509

I came to understand that the drift was not intentional and my assumptions were shared by other engineers: that the schema in vortex should match the schema in MP for that service. So that got me wondering if other services had drift. What this PR does is loop through the stitched services and copy their schema files over to MP and then I ran `yarn dump:local` and committed all the resulting diff - it's a lot!

What should we do about this? @damassi @ansor4 @joeyAghion @dzucconi @anandaroop @bhoggard @sweir27

### CODE REVIEW

File: copy_schemas.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  { name: 'kaws', schema_path: '_schema.graphql' },
+  { name: 'positron', schema_path: 'data/schema.graphql' },
+  { name: 'vortex', schema_path: '_schema.graphql' }
+]
+
+projects.each do |project|
+  target_path = "../#{project[:name]}/#{project[:schema_path]}"
+  data_path = "src/data/#{project[:name]}.graphql"
+  `cp #{target_path} #{data_path}`
+end
```
[the remark this answers is not in the document]
Comment (jonallured) [reply copy_schemas.rb:18]: I just wrote this up quick to see how bad the drift is I'm not saying it should be merged!

**Comment by damassi** [comment]
We use [this](https://github.com/artsy/metaphysics/blob/main/scripts/push-schema-changes.js) to push MP changes to a number of different repos; perhaps we can't go the other way and use it to push stitched schemas to MP, or spin up an intermediary service that will collect these schemas push on a schedule.

**Comment by anandaroop** [comment]
I guess my question is: was it a problem that Vortex's schema was out of date? Should it even be checked into version control (if it's not used locally, but only over in MP)?

Gravity deliberately removed that file from version control a while back (https://github.com/artsy/gravity/pull/13283)

OTOH if we are talking about automating the push from the upstream repos down to MP, then never mind, could be an interesting thing to pursue 👍🏽

**Comment by jonallured** [comment]
> was it a problem that Vortex's schema was out of date?

The event that lead me down this trail was discovering that Vortex had a root query I wanted to use in MP but that it wasn't there. When I looked for code that might have been erasing it from the schema I did not find it. That lead to this Slack thread:

https://artsy.slack.com/archives/CP9P4KR35/p1644856104672509

Which eventually helped me see that the drift wasn't exactly intentional but in any case as the next engineer that wants to make a schema change that affects both Vortex and MP I'm in an awkward position. What seems to be ideal is that the schema for Vortex inside MP match exactly the schema inside Vortex. If we want to massage that schema to ultimately expose something different from MP that's fine but then that should be accounted for in MP.

Do other people have thoughts about this? Yesterday was literally the first day this topic occurred to me so I'm interested in hearing from others!

**Pull request closed without merging**